label_options/caebike-label-options.json

```diff
@@ -1,15 +1,25 @@
 {
   "MODE": [
-    {"value":"walk_run", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
-    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
-    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
-    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
-    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
-    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
-    {"value":"bike_scooter_share", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
-    {"value":"bus_train", "baseMode":"BUS", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.179033},
-    {"value":"other_mode", "baseMode":"OTHER", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0},
-    {"value":"not_a_trip", "baseMode":"UNKNOWN", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0}   
+    {"value":"walk_run", "base_mode": "WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}},
```

Comment:
`met` is given here because it was not the same as the `met` of `WALKING`. This is an override